app/grandchallenge/evaluation/urls.py

```diff
@@ -89,6 +92,21 @@
         PhaseAlgorithmCreate.as_view(),
         name="phase-algorithm-create",
     ),
+    path(
+        "<slug>/interfaces/",
+        AlgorithmInterfacesForPhaseList.as_view(),
+        name="interface-list",
+    ),
+    path(
+        "<slug>/interfaces/create/",
+        AlgorithmInterfaceForPhaseCreate.as_view(),
+        name="interface-create",
+    ),
+    path(
+        "<slug>/interfaces/<uuid:interface_pk>/delete/",
```

Comment:
Nit: shouldn't this be `<slug:slug>` throughout?

Reply:
Sure. We don't do this anywhere else in the evaluation urls. Should I update all of them once I'm here?

Reply:
Done!